src/features/APIdata.js

```diff
@@ -4,6 +4,8 @@ export const APIKey = "951446c9e47a99cb6ccfb776c60023c7"
 
 export const APIPopularMoviesUrl = `${APIUrl}/movie/popular?api_key=${APIKey}`
 
+export const APIPopularPeopleUrl = `${APIUrl}/person/popular?api_key=${APIKey}`
```

Comment:
We need to fix getAPI function in getAPI.js - either add:
``` 
if (!response.ok) {
        throw new Error(response.statusText);
    }
```
or change fetch to axios, because when we have some mistake in url after "https://api.themoviedb.org" part error is not caught :(

Reply:
agree, will add it